common/network/services/udp_sender.cc

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright (c) 2018 IOTA Stiftung
+ * https://github.com/iotaledger/entangled
+ *
+ * Refer to the LICENSE file for licensing information
+ */
+
+#include <boost/asio.hpp>
+
+#include "common/network/logger.h"
+#include "common/network/services/udp_sender.hpp"
+
+bool udp_send(endpoint_t *const endpoint, trit_array_p const hash) {
+  boost::asio::ip::address addr;
+
+  if (endpoint == NULL || endpoint->opaque_inetaddr == NULL) {
+    return false;
+  }
+  size_t trytes_num = num_trytes_for_trits(hash->num_trits);
+  trit_t trytes[trytes_num];
```

Comment:
We have a macro for this?

Reply:
ditto

Reply:
I misread, you're right.

However the type is wrong should be `tryte_t trytes`